setup/web.sh

```diff
@@ -83,6 +83,80 @@ cat conf/mozilla-autoconfig.xml \
 	 > /var/lib/mailinabox/mozilla-autoconfig.xml
 chmod a+r /var/lib/mailinabox/mozilla-autoconfig.xml
 
+# create the MTA-STS policy
+cat << EOF | tee /var/lib/mailinabox/mta-sts.txt
```

Comment:
`tee` echos to the console? I think a simple redirect to the file would work.

Reply:
Fixed in https://github.com/mail-in-a-box/mailinabox/pull/1556/commits/752cf706f572652a743c56bbbd7ead30b2d80fe9